docs/how-to/reef-upgrade.rst

```diff
@@ -0,0 +1,72 @@
+===============
+Upgrade to Reef
+===============
+
+
+Overview
+--------
+
+This guide provides step-by-step instructions on how to upgrade your MicroCeph cluster from the Quincy release to the Reef release. Follow these steps carefully to prevent to ensure a smooth transition.
+
+
+Procedure
+---------
+
+
+Optional but Recommended: Preparation Steps
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Carry out these precautionary steps before initiating the upgrade:
+
+1. **Back up your data**: to avoid any potential data loss, it is recommended to back up all your important data before starting the upgrade.
```

Comment:
I feel we should clarify a bit here on what we are asking them to backup. Backup Cephx keys ? or the actual data stored in the ceph cluster (makes me wonder how).... something like that.

Reply:
This is specifically about the users' data -- this will be application dependent on what to backup. I don't think it's necessary to backup credentials because the snap machinery takes snapshots of those so they should be safe to recover

Reply:
@UtkarshBhatthere if you have ideas on clarifying wording happy to include those :-)

Reply:
I have nothing specific, just reading the doc made me question 'what should I backup ?'. A user may wonder what is at risk of loss on upgrades.

Reply:
I've hopefully clarified a bit